From 46f52d588ab5382a10b68605359e14cc775c86fd Mon Sep 17 00:00:00 2001
From: Dov Grobgeld <dov.grobgeld@gmail.com>
Date: Tue, 24 Jul 2018 21:00:51 +0300
Subject: [PATCH] Fixed bug PFRIBIDI_TYPE_PDF_LRI is undefined. Issue #69

- Fixed typos in symbol definitions when the following condition is
  undefined in fribidi-bidi-types.h:

  #if defined(__FRIBIDI_DOC) || (FRIBIDI_SIZEOF_INT+0 == 4 &&
  !defined(__cplusplus))
---
 lib/fribidi-bidi-types.h | 8 ++++----
 lib/fribidi-bidi.c       | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/lib/fribidi-bidi-types.h b/lib/fribidi-bidi-types.h
index db947a5..d243c5d 100644
--- a/lib/fribidi-bidi-types.h
+++ b/lib/fribidi-bidi-types.h
@@ -223,10 +223,10 @@ typedef uint32_t FriBidiCharType;
 # define FRIBIDI_TYPE_LRO	FRIBIDI_TYPE_LRO_VAL
 # define FRIBIDI_TYPE_RLO	FRIBIDI_TYPE_RLO_VAL
 # define FRIBIDI_TYPE_PDF	FRIBIDI_TYPE_PDF_VAL
-# define FRIBIDI_TYPE_LRI	FRIBIDI_TYPE_PDF_LRI
-# define FRIBIDI_TYPE_RLI	FRIBIDI_TYPE_PDF_RLI
-# define FRIBIDI_TYPE_FSI	FRIBIDI_TYPE_PDF_FSI
-# define FRIBIDI_TYPE_PDI	FRIBIDI_TYPE_PDF_PDI
+# define FRIBIDI_TYPE_LRI	FRIBIDI_TYPE_LRI_VAL
+# define FRIBIDI_TYPE_RLI	FRIBIDI_TYPE_RLI_VAL
+# define FRIBIDI_TYPE_FSI	FRIBIDI_TYPE_FSI_VAL
+# define FRIBIDI_TYPE_PDI	FRIBIDI_TYPE_PDI_VAL
 
 typedef uint32_t FriBidiParType;
 # define FRIBIDI_PAR_LTR	FRIBIDI_TYPE_LTR_VAL
diff --git a/lib/fribidi-bidi.c b/lib/fribidi-bidi.c
index 713fff6..51ec141 100644
--- a/lib/fribidi-bidi.c
+++ b/lib/fribidi-bidi.c
@@ -149,7 +149,7 @@ static FriBidiRun *get_adjacent_run(FriBidiRun *list, fribidi_boolean forward, f
     {
       FriBidiCharType ppp_type = RL_TYPE (ppp);
 
-      if (ppp_type == _FRIBIDI_TYPE_SENTINEL)
+      if (ppp_type == FRIBIDI_TYPE_SENTINEL)
         break;
 
       /* Note that when sweeping forward we continue one run
